ethminer/MinerAux.h

```diff
@@ -272,6 +302,47 @@ class MinerCLI
 		{
 			m_exit = true;
 		}
+		else if ((arg == "-P") && (i + 1 < argc))
+		{
+			string url = argv[++i];
+			URI uri;
+			try {
+				uri = url;
+			}
+			catch (...) {
+				cerr << "Bad endpoint address: " << url << endl;
+				BOOST_THROW_EXCEPTION(BadArgument());
+			}
+			if (!uri.KnownScheme())
+			{
+				cerr << "Unknown URI scheme " << uri.Scheme() << endl;
+				BOOST_THROW_EXCEPTION(BadArgument());
+			}
+			if (m_ep_ix >= k_max_endpoints)
+			{
+				cerr << "Too many endpoints. Maximum is " << k_max_endpoints << endl;
+				BOOST_THROW_EXCEPTION(BadArgument());
+			}
+			m_endpoints[m_ep_ix] = PoolConnection(uri.Host(), uri.Port(), uri.User(), uri.Pswd(), uri.ProtoSecureLevel(), uri.ProtoVersion());
+			
+			OperationMode mode = OperationMode::None;
+			switch (uri.ProtoFamily())
```

Comment:
In my opinion, with next PR (where we have URI in PoolConnection), the PoolManager should get all
PoolClients and then can switch between them, based on PoolConnections ProtocolFamily.

Reply:
That would work.